bowtie/templates/report.html.j2

```diff
@@ -496,16 +509,20 @@
           if (isDisplayed) {
             displayBody.parentElement.removeChild(displayBody);
           } else {
+            {# Add Highlight Row Functionality to clicked accordion only  #}
+            rowHighlightListener(rowClass);
             displayCode(code, targetId);
             const accordionBody = document.querySelector(`#${parentBodyId}`);
             accordionBody.insertBefore(displayBody, accordionBody.firstChild);
           }
         }
       }
 
+      const clipBoardIcon = document.querySelector('.bi-clipboard');
       {# Copy Schema from display #}
       function copyToClipboard(targetTextToCopy, btnId) {
         const textToCopy = document.getElementById(targetTextToCopy).innerText;
+        console.log(clipBoardIcon);
```

Comment:
I'm surprised the linter still doesn't catch this, but you have a stray `console.log` statement left in.

Reply:
Opps! Silly mistake happened!